tests/publiccloud/cloud_netconfig.pm

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: cloud-netconfig-{azure, gc2, gce}
+# Summary: This test ensures the consistency of cloud-netconfig's
+# functionality. The test shall be conducted on a VM in the cloud
+# infrastructure of a supported CSP.
+#
+# This test only contains a minimal functionality that needs to
+# be extended in the future.
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use Test::Assert 'assert_equals';
+use testapi;
+
+sub run {
+    my ( $self, $args ) = @_;
+    my $instance = $args->{my_instance};
+
+    my $cloud_netconfig_svc_status = $instance->ssh_script_run(
+        cmd => 'systemctl status cloud-netconfig.service' );
+    my $cloud_netconf_status_ok = $cloud_netconfig_svc_status == 0
+      || $cloud_netconfig_svc_status == 3 ? 1 : undef;
```

Comment:
```suggestion
      || $cloud_netconfig_svc_status == 3;
```

I think this tenary operator is not needed or is there a reason why we would need `undef` here?

Reply:
Thank you for the comment. I have simplified the code visually, but much shorter would only be possible if the check for the actual status options would be on the SUT via Bash.